crates/bevy_ecs/src/query/default_filters.rs

```diff
@@ -0,0 +1,167 @@
+use crate as bevy_ecs;
+use crate::{component::ComponentId, query::FilteredAccess};
+use bevy_ecs_macros::Resource;
+
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+enum FilterKind {
+    With,
+    Without,
+}
+
+#[derive(PartialEq, Eq, Debug)]
+struct DefaultFilter {
+    component_id: ComponentId,
+    kind: FilterKind,
+}
+
+/// A list of default query filters, these can be used to globally exclude entities from queries.
```

Comment:
I don't think it would be completely obvious to me what this means without an example, but it's not blocking the PR.

A little example inside the docstring with a `Disabled` filter could be useful, to explain that this can be used to affect queries outside of your crate.